app.py

```diff
@@ -169,7 +160,7 @@ class Controller(ViktorController):
 
     viktor_enforce_field_constraints = True  # Resolves upgrade instruction https://docs.viktor.ai/sdk/upgrades#U83
 
-    label = "My Entity Type"
+    label = "WMS map controller"
     parametrization = Parametrization
 
     @WebView("Leaflet sample map", duration_guess=1)
```

Comment:
Saving the leaflet maps can be simplified:

```python
from viktor.core import File
html_result = File()
m.save(html_result.source)
return WebResult(html=html_result)
```

Instead of:

```python
html_result = BytesIO()
m.save(html_result, close_file=False)
return WebResult(html=StringIO(html_result.getvalue().decode("utf-8")))
```

Reply:
done!